util/src/kvdb.rs

```diff
@@ -764,9 +766,11 @@ impl Database {
     pub fn restore(&self, new_db: &str) -> Result<(), UtilError> {
         self.close();
 
+        let mut rng = rand::thread_rng();
+        let backup_name = rng.gen::<u16>().to_string() + "backup_db";
```

Comment:
If I'm not mistaken, the filename of backup database is `37backup_db` or `3543backup_db` or `5675backup_db` ?
Is it too difficult to generate a pretty-print name?
Do you known a function named `mktemp`? 
The same functionality function is not only in `C`, `Java`, `Python`, `Shell`, also in `Rust`.

Reply:
The license of crate 'mktemp' is Mozilla Public License.

Reply:
In most time the `mktemp` is for test. IMO, this method should provide the backup name, and the name spec shoud be defined when we use this method. A better name spec should like `chain-20180801101010010`, be more meaningful.

Reply:
The key point of my opinion is a **better name**.

Reply:
If this is a temporary name, you can use `mktemp` or **other similar functions**.
Otherwise, give it a meaningful name.